pkg/cmd/roachtest/test/test_interface.go

```diff
@@ -26,6 +26,11 @@ const DefaultCockroachPath = "./cockroach"
 // in that environment variable.
 var EnvAssertionsEnabledSeed = "ROACHTEST_ASSERTIONS_ENABLED_SEED"
 
+// EnvWorkloadDurationFlag - evniornment variable to override
+// default run time duration of workload set in tests
+// usage ROACHTEST_PERF_WORKLOAD_DURATION="--duration 5m"
+const EnvWorkloadDurationFlag = "ROACHTEST_PERF_WORKLOAD_DURATION"
```

Comment:
Given the way this is used, we should be able to put any value here. We are also not validating the value. Should we need the environment variable to be specific for duration? This can be a generic flag override for performance specific flags. We also should validate this input.

Reply:
+1, I think if this env variables is only intended for duration purposes, we should have some validations

Reply:
Updated, this is specific to duration, added regex validation for it.
For now the scope is just to duration.